### DISCUSSION TITLE
tell me how to get the whole object { value: 'react', label: 'React' }, and not just the value during onchange

### DISCUSSION THREAD

**Discussion by m1x-developer** [mantinedev/mantine#5104] [Q&A]
<Select
      data={[
        { value: 'react', label: 'React' },
        { value: 'ng', label: 'Angular' },
      ]}
    />
    
    tell me how to get the whole object { value: 'react', label: 'React' }, and not just the value during onchange

**Comment by rtivital** [comment]
`onChange={value => handleChange(yourData.find(item => item.value === value))}`

**Comment by m1x-developer** [comment]
tnx :)
